.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -0,0 +1,110 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Bug Report
+description: Create a report to help us improve Bolt.
+title: "[Bug] "
+labels: ["bug", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > Please search existing issues to avoid duplicates.
+        > For security issues, please contact us privately instead of posting here.
+
+  - type: checkboxes
+    attributes:
+      label: Component Selection
+      description: Which part of Bolt is affected?
+      options:
+        - label: Core Engine (Expression eval, Memory, Vector)
+        - label: Connectors (Hive, Parquet, etc.)
```

Comment:
I don't think `Parquet` belongs with connectors. A separate label for file formats/readers might be worth adding though